src/main/utils/shell-env.ts

```diff
@@ -0,0 +1,67 @@
+import execa from "execa";
```

Comment:
This file is a copy of https://www.npmjs.com/package/shell-env, converted to typescript and added a timeout parameter